src/execute.js

```diff
@@ -108,12 +108,16 @@ export function buildRequest({
         value = parameters
       }
 
-      value = parameter && parameter.name && parameters[parameter.name]
+      value = (parameter && parameter.name && parameters[parameter.name]) || value
```

Comment:
If the `if` above is dead code, should it be removed?  And if it's removed, what value does `value` have before this line?

Also, `paramater` can't be null, can it? If yes, no need to check for `parameter`.

Should rewrite this thing to be consistent with the stuff below.

```
if (condition) {
  value = ...
}
```